WordPress/Classes/Models/WPAccount+Lookup.swift

```diff
@@ -15,6 +15,13 @@ public extension WPAccount {
         return self.uuid == uuid
     }
 
+    // This is here in an extension that belongs to the apps target so we can decouple WPAccount from AppConfiguration.
+    // Decoupling allows moving the type to WordPressData, see https://github.com/wordpress-mobile/WordPress-iOS/issues/24165.
+    @objc
+    static func tokenForUsername(_ username: String) -> String {
+        token(forUsername: username, isJetpack: AppConfiguration.isJetpack)
+    }
+
```

Comment:
Also, @kean I am aware that `AppConfiguration` is due for removal and that `isJetpack` should be removed in favor of an enum. I don't know when that's going to happen, though, so I just kicked the can down the road.

After all, this is a net zero change. I doesn't remove an `AppConfiguration` usage but it doesn't add one either. It simply moves it from one place to another.